libres/lib/include/ert/enkf/local_obsdata_node.hpp

```diff
@@ -40,20 +42,20 @@ active_list_type *
 local_obsdata_node_get_active_list(const local_obsdata_node_type *node);
 active_list_type *
 local_obsdata_node_get_copy_active_list(const local_obsdata_node_type *node);
-void local_obsdata_node_copy_active_list(local_obsdata_node_type *node,
-                                         const active_list_type *active_list);
+extern "C++" void
+local_obsdata_node_copy_active_list(local_obsdata_node_type *node,
+                                    const active_list_type *active_list);
 void local_obsdata_node_add_tstep(local_obsdata_node_type *node, int tstep);
-void local_obsdata_node_add_range(local_obsdata_node_type *node, int step1,
-                                  int step2);
 
 bool local_obsdata_node_tstep_active(const local_obsdata_node_type *node,
                                      int tstep);
 bool local_obsdata_node_all_timestep_active(
     const local_obsdata_node_type *node);
-bool local_obsdata_node_has_tstep(const local_obsdata_node_type *node,
-                                  int tstep);
-void local_obsdata_node_reset_tstep_list(local_obsdata_node_type *node,
-                                         const int_vector_type *step_list);
+extern "C++" bool
```

Comment:
Consider removing extern "C++" and move them outside of extern "C".

Reply:
Never mind, it can't live outside the extern "C" block because it uses the typedef local_obs_node_type that is defined inside the block.